src/engraving/dom/note.cpp

```diff
@@ -1900,7 +1900,15 @@ void Note::setDotRelativeLine(int dotMove)
             return;
         }
     }
-    y *= spatium() * staff()->lineDistance(tick());
+
+    if ((staffType() && staffType()->xmlName() == "stdJianpu")
+        && (staffType() && staffType()->lines() == 0)) {
+        double symWidth = staff()->symWidth(SymId::keysig_1_Jianpu);
+        y = 0.0;
+        y += symWidth / 3.0;
```

Comment:
Also, could you explain what the value 3.0 means here? (I mean, why exactly 3.0?) 
It might be good to add a named constant for it, so that everyone can understand. Like this:
```cpp
        static constexpr double SOME_DESCRIPTIVE_NAME = 3.0;
```